### ISSUE TITLE
wrong capacity and free size of datastore.info against vcsim

### ISSUE THREAD

**Issue by leslie-qiwa** [vmware/govmomi#1121]
If using vcsim as backend and run `govc datastore.info -k`, the capacity and free space are wrong, as below. Seems like `simulator/os_unix.go` already calculate size by MB. 
```
Name:        LocalDS_0
  Path:      /DC0/datastore/LocalDS_0
  Type:      local
  URL:       /var/folders/p1/bx0fgbnj0ljbygmb7hr32zj00000gn/T/govcsim-DC0-LocalDS_0-186611435
  Capacity:  0.2 GB
  Free:      0.1 GB
```

In addition, if VM is created on the datastore, expectation is VM disk size should be counted as well although disk file is not created physically.

I have a working patch, please let me know if I should submit one PR for it or not. Thanks.

**Comment by dougm** [comment]
@leslie-qiwa a PR to fix would be great, thanks.

**Comment by leslie-qiwa** [comment]
Thanks. Will add more tests on my current patch, and submit before next week.

**Comment by dougm** [comment]
Fixed by #1125
